odoo/tools/mail.py

```diff
@@ -599,3 +620,7 @@ def formataddr(pair, charset='utf-8'):
             name = email_addr_escapes_re.sub(r'\\\g<0>', name)
             return f'"{name}" <{local}@{domain}>'
     return f"{local}@{domain}"
+
+def is_common_domain(domain):
```

Comment:
Not sure about the name, it's not merely that they're a *common* domain, it's that they're domains shared by completely unrelated people (not e.g. employees of the same company).